finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,79 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Captain America";
+var specialAbility = "super strength";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Hello " + heroName;
+var catchphrase = `Fear my ${specialAbility}!`;
 
 // Declare two variables - power AND energy - set to integers
+var power = 50;
+var energy = 75;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = power * 500;
+var fullEnergy = energy + 50;
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Adolf Hitler", "Red Skull", "Baron Zemo"];
+var sidekicks = ["Bucky", "Falcon", "Demolition Man"];
 
 // Print the first sidekick to your console
+console.log(sidekicks[0]);
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
```

Comment:
Nice use of bracket notation. A next level here could be to think about a way to print the last element no matter how many elements are in the array.